src/gui/mainwindow.cpp

```diff
@@ -262,7 +309,9 @@ MainWindow::MainWindow(QWidget *parent)
     connect(m_ui->actionIncreasePriority, SIGNAL(triggered()), m_transferListWidget, SLOT(increasePrioSelectedTorrents()));
     connect(m_ui->actionDecreasePriority, SIGNAL(triggered()), m_transferListWidget, SLOT(decreasePrioSelectedTorrents()));
     connect(m_ui->actionBottomPriority, SIGNAL(triggered()), m_transferListWidget, SLOT(bottomPrioSelectedTorrents()));
+#ifndef Q_OS_MAC
     connect(m_ui->actionToggleVisibility, SIGNAL(triggered()), this, SLOT(toggleVisibility()));
```

Comment:
Sorry, but I am not very familiar with Qt, and I do not even have Qt Creator installed.

Quick googling shows me this: https://wiki.qt.io/New_Signal_Slot_Syntax, and by looking at the syntax I am afraid all your code requires such changes, and not just the mac section.

Therefore it is pretty much outside of the scope of this pull request and is better to be done by a Qt dev.